### PR TITLE
Packaging and publishing

### PR DESCRIPTION
**Pull request by yellowhatter**
Work on:
- CI
- Crates.io publishing
- Docker image creation
- Fix crosscompilation issues
- test improvements

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+## Contact
+
+Contact the project developers via the project's "dev" list.
+
+* https://accounts.eclipse.org/mailing-list/zenoh-dev
+
+Or via the Gitter channel.
+
+* https://gitter.im/atolab/zenoh
```
Comment (Mallets) [inline CONTRIBUTING.md:51]: Where did you copy this file from? The link to gitter is very old and everything has now moved to Discord...

Comment (yellowhatter) [reply CONTRIBUTING.md:51]: Oops..... that's the DDS bridge :)
Most of the typical stuff is taken from there

Comment (Mallets) [reply CONTRIBUTING.md:51]: It needs to be update here and also in the zenoh-plugin-dds repository then...

Comment (yellowhatter) [reply CONTRIBUTING.md:51]: I'll do that

Comment (yellowhatter) [reply CONTRIBUTING.md]: mqtt plugin too....

File: CONTRIBUTORS.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# Contributors to Eclipse zenoh-plugin-ros1
+
+These are the contributors to Eclipse zenoh (the initial contributors and the contributors listed in the Git log).
+
+
+| GitHub username | Name                         |
+| --------------- | -----------------------------|
```
Comment (Mallets) [inline CONTRIBUTORS.md:7]: You should add your name here :)

File: Dockerfile [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+# List of installed tools:
+#  * for zenoh-plugin-ros1
+#     - git
+#     - clang
+RUN apt-get update && apt-get -y install git clang build-essential
+
+COPY . .
+# if exists, copy .git directory to be used by git-version crate to determine the version
+COPY .gi? .git/
```
Comment (Mallets) [inline Dockerfile:31]: `.gi?` or `.git`?

**Pull request merged**